internal/database/online/sqlutil/import.go

```diff
@@ -49,8 +50,15 @@ func Import(ctx context.Context, db *sqlx.DB, opt online.ImportOpt, backend type
 			return err
 		}
 		if opt.ExportError != nil {
-			return <-opt.ExportError
+			select {
+			case err := <-opt.ExportError:
+				if err != nil {
```

Comment:
if `opt.ExportError` closed. the case will run, there is a demo:

https://go.dev/play/p/Se1Tw08Kr2J